ts/components/WithTooltip.svelte

```diff
@@ -26,19 +26,27 @@
     export let hideDelay = 0;
 
     let tooltipObject: Tooltip;
-
+    let tooltipOptions: Partial<Tooltip.Options>;
     function createTooltip(element: HTMLElement): void {
         element.title = tooltip;
-        tooltipObject = new Tooltip(element, {
+        tooltipOptions = {
             placement,
             html,
             offset,
             delay: { show: showDelay, hide: hideDelay },
             trigger,
-        });
+        };
+        tooltipObject = new Tooltip(element, tooltipOptions);
     }
 
     onDestroy(() => tooltipObject?.dispose());
+
+    let previousTooltip: string = tooltip;
```

Comment:
`previousTooltip` will always point to the _original_ tooltip. Is that on purpose? If yes, then the wording is a bit confusing (also it could be `const`).

Reply:
Thanks for the heads-up, that was not intended. Like the name suggests, I wanted it to be the previous tooltip. If it always compares with the original tooltip, switching back to the original note type will prevent an update.

Therefore, I won't use a `const`, but rather update `previousTooltip` within the reactive function to make it do what the name suggests.